airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/products_report.py

```diff
@@ -263,6 +280,10 @@ def _get_init_report_body(self, report_date: str, record_type: str, profile):
             if profile.accountInfo.type == "vendor":
                 metrics_list = copy(metrics_list)
                 metrics_list.remove("sku")
+        elif record_type == RecordType.PRODUCTADS and profile.accountInfo.type != "seller":
```

Comment:
Add a unit test for this `if` and assert the removal of `sku` metric. And the same with `display_report.py`.

Reply:
@monai - I've added the unit test to assert the presence of `sku` metric in the report requests based on profiles.